pkg/printers/customcolumn.go

```diff
@@ -151,6 +151,10 @@ type CustomColumnsPrinter struct {
 }
 
 func (s *CustomColumnsPrinter) PrintObj(obj runtime.Object, out io.Writer) error {
+	if internalObjectPreventer.IsForbidden(reflect.Indirect(reflect.ValueOf(obj)).Type().PkgPath()) {
```

Comment:
@deads2k PkgPath was returning an empty string before because runtime.Object is fulfilled by a pointer to an object.
Adding `reflect.Indirect(...` here ensures we always deal with a value and are able to retrieve a non-empty package path